web/components/groups/group-banner-image.tsx

```diff
@@ -0,0 +1,259 @@
+import { PencilIcon, PhotographIcon } from '@heroicons/react/solid'
+import clsx from 'clsx'
+import { Group } from 'common/group'
+import { User } from 'common/user'
+import Image from 'next/image'
+import { useState } from 'react'
+import { updateGroup } from 'web/lib/firebase/groups'
+import { uploadImage } from 'web/lib/firebase/storage'
+import { Button } from '../buttons/button'
+import DropdownMenu from '../comments/dropdown-menu'
+import { Col } from '../layout/col'
+import { Modal, MODAL_CLASS, SCROLLABLE_MODAL_CLASS } from '../layout/modal'
+import { Row } from '../layout/row'
+
+export const DEFAULT_BANNERS = [
+  '/group/default_group_banner_indigo.png',
+  '/group/default_group_banner_green.png',
+  '/group/default_group_banner_red.png',
+]
+
+function isDefaultBanner(url: string) {
+  return DEFAULT_BANNERS.some((path) => path === url)
+}
+export default function BannerImage(props: {
+  group: Group
+  user: User | undefined | null
+  isEditable: boolean
+}) {
+  const { group, user, isEditable } = props
+  const [groupBannerUrl, setGroupBannerUrl] = useState(
+    group.bannerUrl ?? DEFAULT_BANNERS[0]
```

Comment:
If you want a random default banner, you could do:
`DEFAULT_BANNERS[genHash(group.id) % DEFAULT_BANNERS.length]`